proposals/migrate-cli.md

```diff
@@ -0,0 +1,68 @@
+# Migrate Bigchaindb cli for Tendermint
```

Comment:
There's a bigger question here, and that is, will the two products share the same Python code for handling the `bigchaindb` command-line command, or will they have different code for that? I guess your thinking is that they _will_ share the same code. I don't know which is best without attempting implementation.

Reply:
I concur with the idea that it is difficult to comtemplate the effort entailed in either of the approaches. But I guess we could start off with the current code and replace bits and pieces as we move ahead?

Reply:
:+1: